src/site/content/en/blog/wasm-libraries/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: New functionality for developers—brought to you by WebAssembly
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/UgDbKE3G0klQHevSBbQr.jpg
+alt: Low angle photo of a library in Dublin, Ireland
+subhead: |
+  A showcase of tools now available on the web thanks to WebAssembly.
+authors:
+ - nattestad
+date: 2023-04-03
+description: |
+  A showcase of tools now available on the web thanks to WebAssembly.
+tags:
+ - blog
+ - webassembly
+---
+
+WebAssembly enables developers to bring new performant functionality to the web from other languages. Over the past few years developers have really taken advantage of the possibilities. This post showcases just a few of the shiny new tools that you can benefit from, thanks in part to WebAssembly.
+
+## Tools and libraries you can use now
+
+Without further ado, let's jump into the good stuff :D
+
+### SQLite
+
+This complete port of SQLite brings a lightweight, embedded, relational database management system into your hands. To learn more read this [blog post that showcases this incredible port and how to use it](https://developer.chrome.com/blog/sqlite-wasm-in-the-browser-backed-by-the-origin-private-file-system/).
+
+### FFmpeg.wasm
+
+FFmpeg is a free and open-source software project consisting of a suite of libraries and programs for handling video, audio, and other multimedia files and streams. You can find [a wasm compiled version here](https://ffmpegwasm.netlify.app/) ([github repo]()) that lets you do all of this functionality directly in the browser.
```

Comment:
I think we are missing a link to a repo here.

Reply:
Looks like it - https://github.com/ffmpegwasm/ffmpeg.wasm is the one